src/composables/useBcrosLegalApi.ts

```diff
@@ -1,24 +1,48 @@
 /** Returns Legal API URL or Business API GW URL depending on FF. */
+import { LDFlags } from '~/enums/ld-flags'
 
 export const useBcrosLegalApi = () => {
   const config = useRuntimeConfig()
   const { ldInitialized, getStoredFlag } = useBcrosLaunchdarkly()
 
-  let baseURL = config.public.legalApiURL
-  let options = {}
+  function getConfig() {
+    let apiURL = `${config.public.legalApiURL}`
+    let addtionalHeaders = {}
 
-  // Check feature flag to determine which API to use
-  try {
-    if (ldInitialized && getStoredFlag('use-business-api-gw-url')) {
-      baseURL = config.public.businessApiGwUrl
-      options = {
-        headers: { 'X-Apikey': config.public.businessApiKey }
+    // Check feature flag to determine which API to use
+    try {
+      if (ldInitialized && getStoredFlag(LDFlags.UseBusinessApiGwUrl)) {
+        apiURL = `${config.public.businessApiGwUrl}`
+        addtionalHeaders = {
+          'X-Apikey': config.public.businessApiKey
```

Comment:
In the preview URL, this is blank. This is a known issue when using a preview URL.

In other words, this code might work once deployed to Dev. Let me think about whether we should try that... @JazzarKarim , thoughts? (You and I could try this locally first.)

Reply:
PS Don't post the API key, or a screenshot of the key, here.